Service/GithubPubliccodeService.php

```diff
@@ -266,7 +361,7 @@ public function importPubliccodeRepository($repository): ?ObjectEntity
 
         isset($this->io) && $this->io->comment('Checking repository '.$repository['repository']['name']);
         $synchronization->setMapping($repositoriesMapping);
-        $synchronization = $this->synchronizationService->synchronize($synchronization, $repository);
+        $synchronization = $this->synchronizationService->handleSync($synchronization, $repository);
```

Comment:
Is there a reason to revert to handleSync? The synchronize function should replace handleSync in order to force remapping if the mapping changes.